src/controllers/dynakube/token/tokens.go

```diff
@@ -98,14 +99,25 @@ func (tokens Tokens) VerifyValues() error {
 
 func concatErrors(errs []error) error {
 	concatenatedError := ""
+	apiError := 0
```

Comment:
Call it `apiStatus`, wrap 0 in a const like `NoApiError` to give it a meaningful name and reuse it later for checks

Reply:
done